ProjectShop/Makefile

```diff
@@ -0,0 +1,31 @@
+
+migrations:
+	@python manage.py makemigrations
+
+migrate:
+	@python manage.py migrate
+
+app:
+	@python manage.py startapp $(name) /$(name)
+
+test_user:
+	@python manage.py createsuperuser
+
+run:
+	@python manage.py runserver
+
+dbshell:
+	@python manage.py dbshell
+
+shell:
+	@python manage.py shell
+
+test:
+	@python manage.py test
+
+update:
+	@pip install -U -r requirements.txt
+
+collect:
+	@python manage.py collectstatic --settings=$(SETTINGS) --noinput
+
```

Comment:
Maybe it will be a good idea to add commands for docker, as well.

Reply:
Tadey, thank you for your feedback. I appreciate it.
I think it would be better to add docker commands when we actually start using docker.
We might as well add other commands.
Should you have any other ideas, suggestions, or whatever else, feel free to contact me.